src/styles/images/index.md.njk

```diff
@@ -8,53 +8,144 @@ layout: layout-pane.njk
+
+Icons can be more useful in case working systems, where users are familiar with the interface and return to it frequently. In this context, they can help users to scan pages more quickly. In most cases it’s still helpful to include a text label alongside any icons.
+
+Do not use a single icon to represent more than one thing. For example, the search icon (magnifying glass) should only be used for search functionality, and not also for screen magnification.
+
+## Avoid images that contain text
+
+Use written content instead of an image that contains text, wherever possible.
+
+Any text inside an image will be difficult to read, especially for users that need to read text using:
+- screen readers
+- copy and paste
+- highlight
+- resize
+- recolour
+
+However, text can sometimes be an inseparable part of an image, especially in photography. Examples of this might be a photo with a signpost in it, or a company's name on the side of a building.
+
+If you use an image that contains text, you should also include written content nearby that conveys the same meaning.
+
+Only use alt text as a last resort — if you use an image with text that cannot be removed, replaced or supplemented by written content. Be sure to follow the guidance on this page about writing good alt text.
+
+### Avoid images of text
+
+Images made up of text are usually unnecessary. Where possible, you should replace them with alternatives.
+
+Some examples of unnecessary images are:
+- posts from social media
+- excerpts from a document
+- screenshots from presentations
+
+If you do use an image of text, remember to have written content nearby that conveys the same meaning.
+
+#### Exceptions
+
+There are specific types of images where the exact appearance of the text in the image is essential to its purpose. For example, a logo or a handwritten memo.
+
+These types of images must include alt text, but equivalent written content nearby is not required.
+
+However, you should still try to replace it when possible.
+
+### Possible alternatives
+
+If it’s not practical to replace an image with written content, there’s a few ways to replace an image that contains text.
+
+Make sure that the contrast ratio of text colour and all portions of the image that overlap the text [meets level AA of the Web Content Accessibility Guidelines (WCAG 2.1)](https://www.w3.org/TR/WCAG21/#contrast-minimum).
+
+#### Show HTML text over an image
+
+One way to avoid using an image that contains text is to place HTML text over the image instead.
+
+You’ll still need to make sure there is enough contrast, between the text colour and all portions of the image that overlap the text, on all screen sizes.
+
+#### Use a vector graphic image file
+
+If you need to use an image that contains text, create it as a vector graphic such as an SVG file. This ensures that any text stays readable when the user zooms in. Otherwise, text can become pixelated or unreadable.
```

Comment:
```suggestion
You could create the image and text as a vector graphic, such as an SVG file. This ensures that any text stays readable when the user zooms in. Otherwise, text can become pixelated or unreadable.
```